docs/UserGuide.md

```diff
@@ -700,9 +704,13 @@ _Details coming soon ..._
 * Please use a JSON editor software to ensure that the format is still valid after you've edited it.
 
 **I tried to open the closed panel with my mouse, but there is no content displayed when I drag the panel open.**
-* Dictionote is designed to support fast typists, mouse action is limited, please use the `open` command instead.
-
+* Dictionote currently does not allow user to change the visibility of the panel using mouse, 
+  please use the `open` command instead.
 
+**How do I contribute to the project? I have added some new dictionary content and definition.**
```

Comment:
Tiny error, but shouldn't it be *"and definition**s**"* instead?

Reply:
fix